cmd/oaas/handler_build.go

```diff
@@ -110,8 +88,10 @@ func runOsbuild(buildDir string, control *controlJSON, output io.Writer) (string
 	cmd.Dir = buildDir
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		logrus.Errorf("Failed creating result tarball: %v", err)
-		return "", fmt.Errorf("cannot tar output directory: %v, output:\n%s", err, out)
+		err = fmt.Errorf("cannot tar output directory: %w, output:\n%s", err, out)
+		logrus.Errorf(err.Error())
+		mw.Write([]byte(err.Error()))
+		return "", err
 	}
 	logrus.Infof("tar output:\n%s", out)
```

Comment:
I wonder, should we include the tar output in the build log?

Reply:
Thank you